aiohttp_admin/contrib/constants.py

```diff
@@ -0,0 +1,27 @@
+from enum import Enum
+
+
+class ReactComponent(Enum):
```

Comment:
you can do `ReactComponent(str, Enum)` as result `value` in `rc.TEXT_INPUT.value` can be dropped, but this is not critical.